docs/developer/workflows.md

```diff
@@ -14,12 +14,22 @@ Workflows should all be run from the `src/ontology/` directory.
 ---
 
 ## Slurp / migration
-These workflows will determine slurpable / migratable terms. That is, terms that are candidates for integration into Mondo.
+These workflows will determine slurpable / migratable terms. That is, terms that are candidates for integration into 
+Mondo.
+
+#### What determines what is a slurpable / migratable term?
+To be migratable, the term (i) must not already be mapped, (ii) must not be excluded (e.g. not in 
```

Comment:
@sabrinatoro @matentzn I don't know if this is the best place for it, but I added details about what makes a term slurpable / migratable to the documentation.

Reply:
This is great!